tests/ui/consts/const-eval/heap/ptr_made_global_mutated.rs

```diff
@@ -0,0 +1,14 @@
+#![feature(core_intrinsics)]
```

Comment:
```suggestion
//! Ensure that once an allocation is "made global", we can no longer mutate it.
#![feature(core_intrinsics)]
```
Some of the other tests could also use similar comments explaining their purpose.